recipes/sophios/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "sophios" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PolusAI/workflow-inference-compiler/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5c71e369346ba58ccace86897ff8d7cc4dadddb23380af51217a1ca03ecbe41c
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - setuptools >=42
+    - pip 
+    - wheel 
+    - versioneer[toml] ==0.29
```

The PyPI syntax doesn't work on conda recipes. Conda is Python agnostic.

```suggestion
    - versioneer ==0.29
```